index.bs

```diff
@@ -952,6 +953,44 @@ spec: ecma-262; urlPrefix: http://tc39.github.io/ecma262/
   </table>
 </section>
 
+  Navigation {#patch-navigation}
+  ------------------------------
+
+  Patch the <a spec=HTML>navigate</a> algorithm to prevent certain navigations in a
+  portal as follows:
+
+  <div algorithm="navigate patch">
+    In <a spec=HTML>navigate</a>, in the case where <var ignore>resource</var> is a [=request=] whose
+    [=request/url=]'s [=url/scheme=] is "`javascript`", prepend the following step as the first step
+    of the queued task:
+
+    1. If |browsingContext| is a [=portal browsing context=], then [=close a portal
+        element|close=] |browsingContext|'s [=host element=] and abort these steps.
+  </div>
+
+  <div algorithm="process a navigate response patch">
+    In <a spec=HTML>process a navigate response</a>, prepend the following after the initial
+    variable-setup steps:
+
+    1. If |browsingContext| is a [=portal browsing context=], and <var ignore>response</var>'s
+        [=response/url=] is either null or is a [=URL=] whose [=url/scheme=] is not a [=HTTP(S)
+        scheme=], then:
```

Comment:
IIUC this happens after all redirects etc. Do you know if there are any other fetch schemes that could do redirects, and thus would be missed here? Would it matter if they were?

Reply:
Is this the `registerProtocolHandler` case? If so, I'm not sure what we do in Chromium and we should probably file a bug to figure out out.